app/webpacker/controllers/link_controller.js

```diff
@@ -35,4 +36,28 @@ export default class extends Controller {
       }
     });
   }
+
+  prepareChevronOnButtonLinks() {
+    const links = this.contentTarget.querySelectorAll(
+      '.button,.type-description__link'
+    );
+
+    [...links]
+      .filter((link) => link.querySelector('span') === null) // don't touch buttons that already include a span
+      .forEach((link) => {
+        link.innerHTML = this.wrapFinalWordInSpan(
+          link.textContent.trim().split(' ')
+        );
+      });
+  }
+
+  wrapFinalWordInSpan(words) {
+    const wordCount = words.length - 1;
```

Comment:
I think this could be a bit simpler:

```
return words.push(`<span>${words.pop()}</span>`).join(' ');
```

Reply:
Ah, that's much cleaner. Unfortunately JavaScript's `Array.push` returns the new length so it's unchainable; I used `Array.concat` instead.

Reply:
Ah, nice 👍